### ISSUE TITLE
Results from the paper

### ISSUE THREAD

**Issue by DeNeutoy** [jakelever/kindred#14]
Hi, thanks for the great library. I'm one of the authors of [scispacy](https://allenai.github.io/scispacy/) and I was trying to evaluate our base nlp models using the bionlp corpora available through kindred. I've successfully evaluated on the dev set and it gives a nice 2 point boost in F1, but the test set for `2016-BB3-event-test` has no annotations - how did you generate the results in the results directory of this repository? Additionally, it would be great if you could describe the hyperparameter search you ran briefly, so I can try and reproduce it.

Additionally, I tried out the SeeDev corpora but I get the error 
`Kindred doesn't currently support training a classifier with candidate relations that are of more than one type`,
even when using the `2016-SeeDev-binary-*` data.

If I get it working, i'll send a PR which allows users to use scispacy models as a dependency (it just requires a little tweak to the `kindred.Parser` class to allow non-standard spacy models).
Thanks!

**Comment by jakelever** [comment]
Hey, I recently heard about your scispacy work. Very cool stuff. I was wondering about how it'd benefit Kindred so it's great that it sounds like it might help.

So three answers:
- The test sets for the BioNLP shared tasks don't come with relation annotations. We had to save the output and upload them to their online evaluation service. It's to stop people overfitting methods to the test data. I just tried to access it and it looks like its down (link at http://2016.bionlp-st.org/tasks/bb2/bb3-evaluation). Robert Bossy at INRA would be the person to contact about that.
- For the SeeDev issue, I've refactored Kindred a lot since the original paper and had forgotten that SeeDev had overlapping relations (where the same entities are involved in more than one relation). I've fixed this with a MultiLabel classifier for v2.5. Something to note is that the original paper used CoreNLP and I swapped to Spacy for v2 due to nice Python bindings. So it'd be difficult to compare against the original paper results.
- And for hyperparameter tuning, Kindred is designed to work well out-of-the-box. The only tuning it has ever done was a feature selection method from the original paper. But this was clunky and didn't seem to give substantial benefit so it's been removed. I think a reasonable comparison would be spacy versus scispacy with default Kindred parameters for the BB3 and SeeDev tasks (if you can them to bring up the 2016 evaluation service). Alternatively, there are 2019 Shared Tasks about to start with BB3 and SeeDev again (https://2019.bionlp-ost.org/tasks). Not sure how it will be different. Last thought is that a good additional comparison would be how spacy/scispacy performs against CoreNLP. There are new Python bindings for CoreNLP which might make it easier to run a comparison.

Interested to hear your thoughts.
